src/gui/screens/input.py

```diff
@@ -260,6 +273,15 @@ def cb(self, obj, event):
     def get_value(self):
         return self.pin.get_text()
```

Comment:
I think it's more logical to return `None` instead of some custom CANCEL_VALUE here:
```suggestion
        v = self.pin.get_text()
        return None if v == self.CANCEL_VALUE else v
```

`CANCEL_VALUE` is still used internally in the PIN screen, but at least other components don't need to know about it.